app/components/Transaction/index.js

```diff
@@ -8,6 +8,10 @@ import TezosIcon from '../TezosIcon/';
 import { openLinkToBlockExplorer } from '../../utils/general';
 import * as types from '../../constants/TransactionTypes';
 import { READY } from '../../constants/StatusTypes';
+import {
+  REGULAR_CONSUMED_GAS,
```

Comment:
consumed for what, origination, transaction? Also, what's regular? "implicit" accounts are tz1,2,3; originated are kt1.

Reply:
How about we call these

REG_TX_GAS_CONSUMPTION
EMPTY_OUT_TX_GAS_CONSUMPTION

Reply:
ok